src/multiexp.rs

```diff
@@ -328,11 +313,15 @@ fn test_with_bls12() {
             .collect::<Vec<_>>(),
     );
 
+    let now = std::time::Instant::now();
     let naive: <Bls12 as Engine>::G1 = naive_multiexp(g.clone(), v.clone());
+    println!("Naive: {}", now.elapsed().as_millis());
 
+    let now = std::time::Instant::now();
     let pool = Worker::new();
 
     let fast = multiexp(&pool, (g, 0), FullDensity, v_bits).wait().unwrap();
+    println!("Fast: {}", now.elapsed().as_millis());
```

Comment:
These prints are suppressed, I think we try to avoid suppressed messages being printed in tests in our other crates right?

Reply:
Yeah. Removed.